src/Driver.zig

```diff
@@ -72,6 +74,7 @@ pub const usage =
     \\                          Disallow '$' in identifiers
     \\  -fmacro-backtrace-limit=<limit>
     \\                          Set limit on how many macro expansion traces are shown in errors (default 6)
+    \\  -fnative-half-type      Use the native half type for __fp16 instead of promoting to float
```

Comment:
Neither clang nor GCC seem to expose this option, should we?

Reply:
It's hidden behind `-Xclang`  https://godbolt.org/z/x739v9z7v - do you think we should have a similar flag for non-standard options like that? I don't think GCC has an option for native half. There's also a similar `-Xclang` flag `-fnative-half-arguments-and-returns` which allows `__fp16` as an argument/return type on all platforms

Reply:
Ah, I didn't realize that's how they do it. I guess this is fine for now then?